### PR TITLE
Update schema prompts to inquerier

### PR DESCRIPTION
**Pull request by macmv**
Ticket(s): ENG-5602

This one is easier to read commit-by-commit.

It updates the prompts to use inquirer, and fixes database URLs that end in a slash, by constructing them with a `new URL`.

### CODE REVIEW

File: src/commands/schema/diff.ts [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+import fetch from "node-fetch";
 
-class DiffSchemaCommand extends SchemaCommand {
+export default class DiffSchemaCommand extends SchemaCommand {
   static flags = {
     ...SchemaCommand.flags,
   };
 
+  static description = "Print the diff between local and remote schema.";
+  static examples = ["$ fauna schema diff --dir schemas/myschema"];
```
Comment (fauna-chase) [inline src/commands/schema/diff.ts:10]: should we show an example without the --dir as well?

File: src/commands/schema/push.ts [hunk 2]
```diff
@@ -11,28 +12,18 @@ class PushSchemaCommand extends SchemaCommand {
...
-    }
-    if (["no", "n"].includes(resp.toLowerCase())) {
-      return false;
-    }
-    console.log("Please type 'yes' or 'no'");
-    return this.confirm(msg);
-  }
+  static description = "Push the current project's .fsl files to Fauna.";
+
+  static examples = ["$ fauna schema push --dir schemas/myschema"];
```
Comment (fauna-chase) [inline src/commands/schema/push.ts:17]: same question with --dir here

File: src/commands/schema/push.ts [hunk 3]
```diff
@@ -52,17 +43,23 @@ class PushSchemaCommand extends SchemaCommand {
...
           this.log(`Proposed diff:\n`);
           this.log(json.diff);
         } else {
           this.log("No logical changes.");
-          msg = "Push file contents anyway?"
+          message = "Push file contents anyway?";
         }
-        if (await this.confirm(msg)) {
+        this.log(`Proposed diff:\n`);
+        this.log(json.diff);
```
Comment (fauna-chase) [inline src/commands/schema/push.ts:55]: are we logging the diff twice here?

**Review by fauna-chase — APPROVED** [review]

File: src/commands/schema/diff.ts (hunk 1 above)
Comment (macmv) [reply src/commands/schema/diff.ts:10]: yeah we should

File: src/commands/schema/push.ts (hunk 3 above)
Comment (macmv) [reply src/commands/schema/push.ts:55]: oh yeah oops. let me go check, i think i messed up the rebase

**Comment by fauna-chase** [comment]
o also, thank you for moving these all to type script :)

**Comment by macmv** [comment]
oh but of course. it gives me nice auto-completions :)

**Pull request merged**